xontrib_avox_poetry/venv_poetry.py

```diff
@@ -4,11 +4,9 @@
 from pathlib import Path
 from xonsh.built_ins import XSH
 from .utils import deep_get
-
+from xonsh.lib.subprocess import check_output
```

Comment:
this sometimes creates issue regarding threads and capturing. It is better to use `subprocess.check_output(..., env=XSH.env.detype() )`